Marlin/Marlin_main.cpp

```diff
@@ -5452,6 +5450,17 @@ inline void gcode_M42() {
           SERIAL_PROTOCOLLNPGM(".  DEPLOYED state: LOW (logic 0)");
           SERIAL_PROTOCOLLNPGM(".  STOWED (triggered) state: HIGH (logic 1)");
         }
+        #if !defined(BLTOUCH)
+          SERIAL_ERROR_START;
+          SERIAL_ERRORLNPGM("BLTOUCH is enabled but is NOT compatible with TLTouch");
+          SERIAL_ERRORLNPGM(". Make sure the following lines are in Configuration.h");
+          SERIAL_ERRORLNPGM(".   //#define BLTOUCH");
+          SERIAL_ERRORLNPGM(".   #define Z_ENDSTOP_SERVO_NR 0");
+          SERIAL_ERRORLNPGM(".   #define Z_SERVO_ANGLES {10,90}");
+          SERIAL_ERRORLNPGM(".   #define Z_MIN_ENDSTOP_INVERTING true");
+          SERIAL_ERRORLNPGM(".   #define Z_MIN_PROBE_ENDSTOP_INVERTING true");
```

Comment:
This is too verbose to include in the firmware. We need to start limiting the number of strings we're throwing in, as it further constrains which lower-capacity AVR chips we can support and how many features these less roomy AVRs can use before they run out of space. At most, refer users to the documentation.